### ISSUE TITLE
After assigning a text color - we need to get back to the default color

### ISSUE THREAD

**Issue by q2apro** [samclarke/SCEditor#329]
When you select a text color and write the word, then want to deselect it, there is no option: 

```
<p>
    Let us start to write. Now we assign a color before we write the colored word: <br />
</p>
<p>
    <span style="color: rgb(255, 0, 0);">red is assigned, let's deselect it<span style="color: rgb(0, 0, 0);"> oh I had to choose black as there is no &quot;default color&quot; option!</span></span><br />
</p>
```

Instead of the deselect - which is missing - I have to select the `black color` as default (black matches my default CSS color). This creates several color spans for black that are unnecessary.

I suggest to add a feature that CKEditor calls "automatic", we could call it "default": 

![example](https://cloud.githubusercontent.com/assets/6764548/3863164/4afa5d86-1f45-11e4-9b6c-46a2f748ef03.png)

It is important to prevent all the `<span style="color: rgb(0, 0, 0);">`. In a long text of one of my users there were about 20 occurences of this black-span.

**Comment by q2apro** [comment]
Push: Is there any solution in sight?

**Comment by Cu8eeeR** [comment]
@samclarke - should be fixex ASAP, quite annoying!

**Comment by samclarke** [comment]
This seems to be two issues in one. The duplicate span tags issue is fixed by #804 and I've made a new issue for the automatic colour option (#858).
